crates/sparrow-runtime/src/execute/operation/merge.rs

```diff
@@ -690,19 +690,19 @@ mod tests {
         let operation_0 = batch_from_csv(
             "
         _time,_subsort,_key_hash,e0
-        2000,0,1,1
-        3000,1,1,2
-        4000,0,2,3",
+        1970-01-01T00:00:00.000002000,0,1,1
+        1970-01-01T00:00:00.000003000,1,1,2
+        1970-01-01T00:00:00.000004000,0,2,3",
             None,
         )
         .unwrap();
         let operation_1 = batch_from_csv(
             "
         _time,_subsort,_key_hash,e0,e2
-        2500,0,2,1,3
```

Comment:
The failing tests seem to have been caused by the CSV parser no longer accepting integers as timestamps. I don't believe this will affect user CSV files, since prepare should parse these as numbers and then cast the numbers, which will be a different path than trying to parse straight to timestamps.